.github/actions/get-issue-url-and-version-number-v1/src/run.ts

```diff
@@ -0,0 +1,44 @@
+import type { Core, GitHub, Issue } from './types'
+import { getExecOutput } from '@actions/exec'
+
+export default async function run(core: Core, github: GitHub) {
+  const { repo, owner } = github.context.repo
+  try {
+    const {
+      stdout: rawIssue,
+      stderr: issueError,
+      exitCode: issueExitCode
+    } = await getExecOutput(
+      `gh issue list --repo ${owner}/${repo} --label release --state open --json url,title`
+    )
+
+    if (issueExitCode) {
+      throw new Error(`Error getting issue:\n${issueError}`)
+    }
+
+    const [issue] = JSON.parse(rawIssue.trim()) as Issue[]
+    core.info(`Found issue: ${JSON.stringify(issue, null, 2)}`)
+
+    const issueUrl = issue.url
+    core.info(`Issue URL: ${issueUrl}`)
+
+    const issueTitle = issue.title
+    core.info(`Issue title: ${issueTitle}`)
+
+    // get version number from issue title (e.g. "1.33.7")
+    const versionNumber = issueTitle.match(/\d+.\d+.\d+/)
```

Comment:
Need to escape the periods

```suggestion
    const versionNumber = issueTitle.match(/\d+\.\d+\.\d+/)
```